### DISCUSSION TITLE
Change default plot_trajectory() behavior?

### DISCUSSION THREAD

**Discussion by jenniferyee** [rpoleski/MulensModel#101] [Ideas]
1. Currently `model.plot_trajectory()` has a default caustics=False. I think for n_lenses > 1, the default should be caustics=True. Otherwise, it always takes extra work to plot the trajectory for the default case: 2L1S.
2. The way that the arrow is defined in `model.plot_trajectory()` is annoying. I frequently find it is huge, especially when I zoom in to see the detailed behavior near a caustic. Instead, the size should be scaled to the size of the window, e.g. as a fraction of the plot axis.

**Comment by rpoleski** [comment]
1. We should change default to `None` and then decide `True`/`False` depending on the number of lenses. I'm guessing that originally, we thought to save on calculations.
2. Plotting as a fraction of plot axis won't solve the issue fully, because one can change that before calling `plt.show()` or similar. I think the arrow plotting worked fine some time ago on test cases. Maybe some default values in matplotlib changed since then.

**Reply by jenniferyee** [reply]
The arrow size is defined by x[i], x[i+1], so large arrows can occur when the time steps are large, such as when plotting the source trajectory for a wide-orbit planet.

**Comment by rpoleski** [comment]
I guess we can calculate ranges: `max(x)-min(x)` and `max(y)-min(y)`. Then used these to set the size of the arrow.
